sources/web/datalab/polymer/components/datalab-sidebar/datalab-sidebar.html

```diff
@@ -91,11 +91,19 @@
         <a href="/sessions" id="sessions">
           <paper-button class="sidebar-button">
             <div class="sidebar-button-container">
-              <iron-icon class="sidebar-button-icon" icon="av:featured-play-list"></iron-icon>
+              <iron-icon class="sidebar-button-icon" icon="list"></iron-icon>
               <div class="sidebar-button-title">Sessions</div>
             </div>
           </paper-button>
         </a>
+        <a href="/terminal" id="terminal">
```

Comment:
The new UI doesn't support basepath yet, I'll still do a pass over all these to fix them when I get the basepath setting from the server. That'll be a separate change.